src/PlaywrightSharp/PlaywrightSharp.csproj

```diff
@@ -54,11 +54,27 @@
       <InProject>false</InProject>
     </DriverFiles>
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="BrowserContext.old.cs" />
+    <Compile Remove="__TRASH\BrowserContext.old.cs" />
+    <Compile Remove="__TRASH\IBrowserContext.cs" />
+    <Compile Remove="__TRASH\NetworkCookie.cs" />
+    <Compile Remove="__TRASH\SetNetworkCookieParam.cs" />
+    <Compile Remove="__TRASH\TimeoutSettings.cs" />
+  </ItemGroup>
```

Comment:
we don't need this

Reply:
BLARGH, I was so focused on getting this through the door I didn't clean it up. Thanks for reminding me!